src/styles/components/_header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-family: "Roboto", sans-serif;
```

Comment:
add please alternative `font-family`, like `font-family: "Roboto", "Helvetica", sans-serif;`

Reply:
Did that for .page component